src/code/views/inspector-panel-view.coffee

```diff
@@ -2,31 +2,67 @@ NodeInspectorView = React.createFactory require './node-inspector-view'
 LinkInspectorView = React.createFactory require './link-inspector-view'
 PaletteInspectorView = React.createFactory require './palette-inspector-view'
 
-{div, i} = React.DOM
+{div, i, span} = React.DOM
 
-module.exports = React.createClass
+ToolButton = React.createFactory React.createClass
+  displayName: 'toolButton'
+  render: ->
+    name = @props.name
+    onClick = =>
+      @props.onClick(@props.name) if @props.onClick
 
-  displayName: 'InspectorPanelView'
+    classes = "icon-#{name} tool-button"
+    classes = "#{classes} selected" if @props.selected
+    classes = "#{classes} disabled" if @props.disabled
+    (div {className: classes, onClick: onClick})
 
-  getInitialState: ->
-    expanded: true
+ToolPanel = React.createFactory React.createClass
+  displayName: 'toolPanel'
 
-  collapse: ->
-    @setState {expanded: false}
+  buttonData: [
+      {name: "plus",  shows: "add"}
+      {name: "brush", shows: "design"}
+      {name: "ruler", shows: "values"}
+      {name: "curve", shows: "relations"}
+    ]
 
-  expand: ->
-    @setState {expanded: true}
+  buttonProps: (button) ->
+    name: button.name
+    shows: button.shows
+    selected: @props.nowShowing is button.shows
+    onClick: =>
```

Comment:
I think a title to show on hover should also be added - I was confused about what the new buttons did.